### ISSUE TITLE
Babel / Traceur NodeJS resolution bug

### ISSUE THREAD

**Issue by arcanis** [systemjs/systemjs#380]
It seems that babel / traceur are not correctly recognized by Systemjs (even when installed as peers).

Here is what I did :

```
arcanis@~/project/tests # ls
_runner.js  test.js

arcanis@~/project/tests # npm install systemjs traceur babel-core babel
...

arcanis@~/project/tests # cat _runner.js
var System = require( 'systemjs' );

System.transpiler = 'babel';

System.import( process.argv[ 2 ] ).then( function ( m ) {
    return m.test( );
} ).catch( function ( error ) {
    console.error( error.stack || error );
} );

arcanis@~/project/tests # node ./_runner.js test
Error: Error loading "test" at file:/home/arcanis/project/tests/test.js
Error loading "babel" at file:/home/arcanis/project/tests/babel.js
ENOENT, open '/home/arcanis/project/tests/babel.js'

arcanis@~/project/tests #
```

**Comment by guybedford** [comment]
Thanks, yes this is an issue. We have resolution in es6-module-loader, but that doesn't carry through here.

The quick fix is just to do:

```javascript
System.paths['babel'] = require.resolve('babel-core/browser.js');
```

**Comment by guybedford** [comment]
Fixed in 0.16.2.
